### PR TITLE
Feature/799 non attains tribes

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [HMW-799](https://jira.epa.gov/browse/HMW-799)

## Main Changes:
* Removed the restriction on the Tribal page to only show tribes that are in the ATTAINS control table.
* Added a separate WQX tribal mapping list for tribes that _only_ have data in WQX. This list is merged with the ATTAINS list at runtime.
* Added the Past Water Conditions toggle table and date slider to the Tribal page. This is currently placed just below the map, but this may be moved depending on feedback.

## Steps to Test:
1. Go to http://localhost:3000/tribe/KNRD_WQX. Ensure the location displays with mapped monitoring locations, and with the date slider and toggle table below the map. This is a location from the `wqxTribeMapping.json` list.
2. Go to http://localhost:3000/tribe/KAWNATON, and repeat the checks from step 1. This is a location from the `attainsTribeMapping.json` list that does not exist in the ATTAINS control table, but has WQX data.
3. Select various tribes from the drop-down list while changing the Past Water Conditions filters. Verify the filters reset, both on the map and in the filter components.
4. Go to http://localhost:3000/community/dc/monitoring. Check that the Past Water Conditions filters still work as they did previously.

### CODE REVIEW

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Looks good! I like that the past water conditions is being moved into a separate component.

I'm seeing an issue where the numbers next to the switches aren't being updated. The totals are correct though. 

Local:
image

Prod:
image

File: app/client/src/components/pages/StateTribal.tsx [hunk 1]
```diff
@@ -171,18 +186,41 @@ function StateTribal() {
...
+            ...acc[cur.epaId],
+            wqxIds: [...acc[cur.epaId].wqxIds, ...cur.wqxIds],
+          };
+        }
+        return acc;
+      }, {}),
+    ).map((tribe) => ({
+      ...tribe,
+      wqxIds: Array.from(new Set(tribe.wqxIds)),
+    }));
```
[the remark this answers is not in the document]
Comment (maxdiebold-erg) [reply app/client/src/components/pages/StateTribal.tsx]: Good call!

**Comment by maxdiebold-erg** [comment]
@cschwinderg Good catch on the toggle table counts, I must have only been looking at the totals row. That should be fixed now.

**Review by cschwinderg — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**